frontend/src/pages/clusterSettings/CullerSettings.tsx

```diff
@@ -0,0 +1,184 @@
+  initialValue: number;
+  cullerTimeout: number;
+  setCullerTimeout: (timeout: number) => void;
+};
+
+const CullerSettings: React.FC<CullerSettingsProps> = ({
+  initialValue,
+  cullerTimeout,
+  setCullerTimeout,
+}) => {
+  const [cullerTimeoutChecked, setCullerTimeoutChecked] =
+    React.useState<string>(CULLER_TIMEOUT_UNLIMITED);
+  const [hour, setHour] = React.useState(DEFAULT_HOUR);
+  const [minute, setMinute] = React.useState(0);
+
+  React.useEffect(() => {
+    setCullerTimeout(initialValue);
+    if (initialValue !== DEFAULT_CULLER_TIMEOUT) {
+      setCullerTimeoutChecked(CULLER_TIMEOUT_LIMITED);
+      setHour(getHourAndMinuteByTimeout(initialValue).hour);
+      setMinute(getHourAndMinuteByTimeout(initialValue).minute);
+    }
+  }, [initialValue, setCullerTimeout]);
+
+  React.useEffect(() => {
+    if (cullerTimeoutChecked === CULLER_TIMEOUT_UNLIMITED) {
+      setCullerTimeout(DEFAULT_CULLER_TIMEOUT);
+    } else if (cullerTimeoutChecked === CULLER_TIMEOUT_LIMITED) {
+      setCullerTimeout(getTimeoutByHourAndMinute(hour, minute));
+    }
+  }, [hour, minute, cullerTimeoutChecked, setCullerTimeout]);
+
+  const radioCheckedChange = (_, event) => {
+    const { value } = event.currentTarget;
+    setCullerTimeoutChecked(value);
+  };
+
+  return (
+    <SettingSection
+      title="Stop idle notebooks"
+      description="Set the time limit for idle notebooks to be stopped."
+      footer={
+        <HelperText>
+          <HelperTextItem>
+            All idle notebooks are stopped at cluster log out. To edit the cluster log out time,
+            discuss with your OpenShift administrator to see if the OpenShift Authentication Timeout
+            value can be modified.
+          </HelperTextItem>
+        </HelperText>
+      }
+    >
+      <Stack hasGutter>
+        <StackItem>
+          <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsSm' }}>
+            <FlexItem>
+              <Radio
+                id="culler-timeout-unlimited"
+                data-id="culler-timeout-unlimited"
+                label="Do not stop idle notebooks"
+                isChecked={cullerTimeoutChecked === CULLER_TIMEOUT_UNLIMITED}
+                name={CULLER_TIMEOUT_UNLIMITED}
+                onChange={radioCheckedChange}
+                value={CULLER_TIMEOUT_UNLIMITED}
+              />
+            </FlexItem>
+            <FlexItem>
+              <Radio
+                id="culler-timeout-limited"
+                data-id="culler-timeout-limited"
+                label="Stop idle notebooks after"
+                isChecked={cullerTimeoutChecked === CULLER_TIMEOUT_LIMITED}
+                name={CULLER_TIMEOUT_LIMITED}
+                onChange={radioCheckedChange}
+                value={CULLER_TIMEOUT_LIMITED}
+                body={
+                  <InputGroup>
+                    <TextInput
+                      id="hour-input"
+                      style={{ maxWidth: '60px' }}
```

Comment:
I've asked if there's a way of controlling the style of a textinupt outside of overriding the maxWidth, but in the meantime I think it's fine